lightly/models/modules/heads.py

```diff
@@ -215,3 +215,42 @@ def __init__(self,
             (input_dim, hidden_dim, nn.BatchNorm1d(hidden_dim), nn.ReLU()),
             (hidden_dim, output_dim, None, None),
         ])
+
+
+class SwaVProjectionHead(ProjectionHead):
+    """Projection head used for SwaV.
+
+    "We implement in SwAV the improvements used in SimCLR, i.e. (...) and the
+    MLP projection head." [0]
+
+    [0]: SwaV, 2020, https://arxiv.org/abs/2006.09882
+
+    """
+    def __init__(self,
+                 input_dim: int,
```

Comment:
default parameters from paper?

Reply:
we don't have them for any of the other heads